examples/.config/model_params_pytorch.json

```diff
@@ -586,6 +586,20 @@
       "batch_size": 1,
       "main_script": "eval_lambada.py"
     },
+    "opt-30b_sq": {
+      "model_src_dir": "nlp/huggingface_models/language-modeling/quantization/ptq_static/ipex/smooth_quant",
+      "dataset_location": "",
+      "input_model": "facebook/opt-30b",
```

Comment:
opt-30b needs outlier suppression,  the existing code should get near 0.0 accuracy.

Reply:
@hshen14 should we remove this model from the main page?